bin/cij_tlint

```diff
@@ -276,9 +277,9 @@ def main(roots):
 
 if __name__ == "__main__":
 
-    CIJ_ROOT = os.environ.get("CIJ_ROOT")
-    if not (CIJ_ROOT and os.path.exists(CIJ_ROOT)):
-        print("Please set ENV. VAR. CIJ_ROOT, correctly")
+    CONF = cij.conf.from_system()
+    if CONF is None:
+        cij.err("rprt: failed retrieving CIJOE configuration")
```

Comment:
Is there a reason this is called `CONF` here? I think it's called `conf` in other places

Reply:
Yeah, it is in global-scope, some PEP says that variables in global-scope must be UPPERCASE.

Reply:
Ah, I didn't realize that -- cool!